### PR TITLE
docs(readme): fix typos, add summary of all commands

### PR DESCRIPTION
**Pull request by anshumanv**
fix typos, add summary of all commands in commands section

**What kind of change does this PR introduce?**
docs

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
Yes

**Summary**
* fix some typos/grammars
* Add a quick summary of all commands in readme.

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
   - [How to install](#how-to-install)
 * [Getting Started](#getting-started)
 * [webpack CLI Scaffolds](#webpack-cli-scaffolds)
-* Commands
+* [Commands](#commands)
   - [`webpack-cli init`](./packages/init/README.md#webpack-cli-init)
```
Comment (anshumanv) [inline README.md]: Should we still keep all list of commands here? 🤔

Comment (misterdev) [reply README.md]: I like the new one, I would remove this 👍

File: README.md [hunk 2]
```diff
@@ -52,22 +52,32 @@ Otherwise `npm install --save-dev webpack-cli` or `yarn add webpack-cli --dev` w
...
-When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed! Otherwise, you would need to install webpack CLI and the packages you want to use. If we want to use the `init` functionality to create a new `webpack.config.js` configuration file:
+When you have followed the [Getting Started](https://webpack.js.org/guides/getting-started/) guide of webpack then webpack CLI is already installed! Otherwise, you would need to install webpack CLI and the packages you want to use. If we want to use the `init` command to create a new `webpack.config.js` configuration file:
 
 ```sh
 npm i webpack-cli @webpack-cli/init
 npx webpack-cli init
 ```
 
-You will answer many questions when running the `init` so webpack CLI can provide the best fitting configuration.  
+You will be prompted for several questions about what how you want to generate your config file when running the `init` command so webpack CLI can provide the best fitting configuration.
```
Comment (misterdev) [inline README.md]: "Some" instead of "several"/"many" sounds better to me :)

File: README.md (hunk 1 above)
Comment (anshumanv) [reply README.md]: Removing, let's see how everyone else finds it :)

File: README.md (hunk 2 above)
Comment (anshumanv) [reply README.md]: Yes, updated 👍

**Comment by anshumanv** [comment]
@misterdev since (utils, generators, webpack-scaffold) packages are just helpers to other packages and don't have a real command so I'm not sure if it's the best idea to list them in the `commands` section. WDYT? :)

**Comment by misterdev** [comment]
Yeah, I would make another list e.g. "Packages" after the "Commands" section :)
Another idea I like is to have a structure like this:

**Packages**
We organize webpack CLI as a multi-package repository. Every command has a dedicated subfolder in the packages Folder [...]
- Commands
  Supporting developers is an important task for webpack CLI. Thus, webpack CLI provides different 
  commands for many common tasks
  ... list of all commands with description
- Utilities
  ... list of all other packages with descriptions

IMO, having a complete overview of the repository in the REAME is something desirable

PS: I can work on this, in case it is not in the scope of this PR or you have something else to do :)

**Comment by anshumanv** [comment]
Sounds good @misterdev, just a trivial change. ✨

**Pull request merged**